src/test/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/uiTest/pages/SettingsFrame.kt

```diff
@@ -0,0 +1,165 @@
+
+    // Action for separator of coverage
+    val generalSettingsSeparator
+        get() = actionLink(byXpath("//div[@class='JXTitledSeparator'][.//div[@text='General settings']]"))
+
+    // Action for search algorithm label
+    val searchAlgorithmLabel
+        get() = jLabel(byXpath("//div[@text='Select search algorithm']"))
+
+    // Action for search algorithm combo box
+    val searchAlgorithmComboBox
+        get() = comboBox(byXpath("//div[@class='ComboBox']"))
+
+    // Action for seed label
+    val seedLabel
+        get() = jLabel(byXpath("//div[@text='Seed(random if left empty) ']"))
+
+    // Action for configuration id label
+    val configurationIdLabel
+        get() = jLabel(byXpath("//div[@text='Select configuration id (null if left empty) ']"))
+
+    // Action for Execute tests in a sandbox environment checkbox
+    val executeTestsCheckbox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Line coverage']"))
+
+    // Action for Debug mode checkbox
+    val debugModeCheckbox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Line coverage']"))
+
+    // Action for Minimize test suite after generation checkbox
+    val minimiseTestSuiteCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Line coverage']"))
+
+    // Action for Flaky tests checkbox
+    val flakyTestCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Line coverage']"))
+
+    // Action for separator of coverage
+    val coverageSeparator
+        get() = actionLink(byXpath("//div[@class='JXTitledSeparator'][.//div[@text='Criterion selection']]"))
+
+    // Action for Line coverage checkbox
+    val lineCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Line coverage']"))
+
+    // Action for Branch coverage checkbox
+    val branchCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Branch coverage']"))
+
+    // Action for Exception coverage checkbox
+    val exceptionCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Exception coverage']"))
+
+    // Action for Mutation coverage checkbox
+    val mutationCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Mutation coverage']"))
+
+    // Action for Output coverage checkbox
+    val outputCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Output coverage']"))
+
+    // Action for Method coverage checkbox
+    val methodCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Method coverage']"))
+
+    // Action for Method no exception coverage checkbox
+    val methodNoExcCoverageCheckBox
+        get() = checkBox(byXpath("//div[@class='JCheckBox' and @text='Method no exception coverage']"))
+
+    // Action for CBranch coverage checkbox
+    val cBranchCoverageCheckBox
```

Comment:
And I assume you get the `div` from localhost?

Reply:
Yes